charts/libsql/values.yaml

```
@@ -0,0 +1,79 @@
+volumeMounts:
+ - name: data
+   mountPath: "/var/libsql"
+   readOnly: false
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+# Set to true to disable the builtin antiAffinity which will try to place 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)